linux_os/guide/system/permissions/restrictions/sysctl_kernel_unprivileged_bpf_disabled/rule.yml

```diff
@@ -1,8 +1,8 @@
 documentation_complete: true
 
-prodtype: rhel8,fedora,ol8
+prodtype: rhel7,rhel8,fedora,ol8
```

Comment:
Do you know in which RHEL7 profile this rule should be selected? `ospp` problably?
Also, add the entry in the RHEL7's `sysctl_values.csv` file.